Playbooks/Zscaler-add-Domains-to-URL-Category/README.md

```diff
@@ -63,113 +50,163 @@ Hover over the **Activation** button and click **Activate** to enable the new Ro
-Enter a name for the account and in the drop-down box below that select the Role you created in the previous step.
+Enter a **Login ID** for the API administrator account you want to create. 
+
+In the **Email** box you can enter a preexisting service account, or simply make up an email address which you will not use in your domain. There is no need for email access for this account.
+
+Enter a name for the account and in the drop-down box below, select the Role you created in the previous step.
+
 Make sure **Password Based Login** is checked and create a secure password for this API account.
-The **email address and password** will be required when deploying the ARM template.
-Click on **Save** then hover over the **Activation** button and click **Activate** to enable the new Administrator Account.
+
+Take note of the email address and password, as it will be needed during deployment.
 
 ![](Images/zgit6.png)
 
+Click **Save**, then hover over the **Activation** button and click **Activate**. This will enable the new administrator account.
+
 ![](Images/Activate.png)
 
-To get your **API key** hover over the **Administration** button and click on **API Key Management**
+#### Zscaler API Key:
+
+To get your **API key** hover over the **Administration** button and click on **API Key Management**.
 
 ![](Images/zgit7.png)
 
-Here you will find you **API Key** as well as **Zscaler Instance Name.** You will need **both** when deploying the ARM template.  The typical instance names are Zscaler, ZscalerOne, ZscalerTwo, ZscalerThree, and ZsCloud.
+Here you will find you **API Key** as well as **Zscaler Instance Name.** You will need both of these when deploying this playbook. The typical instance names are Zscaler, ZscalerOne, ZscalerTwo, ZscalerThree, and ZsCloud.
 
 ![](Images/zgit8.png)
 
-The last thing we will need from the Zscaler Portal is the Custom URL category. Hover over the **Administration** button and click on **URL Categories.** You will need to copy down the **exact** name of the Custom URL category you want the domains from the Sentinel Incident added to. In the example below we use name of our Custom category AS_Blocklist. This category in our test environment configured to a Zscaler access policy that disallows users access to any domain in in that list.
+#### Zscaler URL Category:
+
+Lastly, you will need to note the custom URL category you want the domains from Microsoft Sentinel incidents added to. 
+
+Hover over the **Administration** button and click on **URL Categories.** 
+
+Take note of the name of your desired URL category, as it will be needed during deployment. 
+
+In the example below we use name of our Custom category AS_Blocklist. 
+This category in our test environment configured to a Zscaler access policy that disallows users access to any domain in in that list.
 
```

Comment:
Can you make these steps bulleted for readability?

Reply:
I have added bullets for the actionable steps in each section. However, throughout the entirety of our documentation, we reserve bulleted lists almost exclusively for tracking requirements, not the more granular steps. Since nearly the entirety of the documentation is actionable steps, I'm worried these changes are a bit gratuitous, however, if they look better to you, we can keep them.

Reply:
@AcceleryntSecurityDev, I was following the document as end user and believe me, now that you have added the bullets, I find it more readable and easier to follow.